.github/workflows/ci.yml

```diff
@@ -26,7 +26,44 @@ jobs:
         run: go install github.com/magefile/mage
 
       - name: Check Vulnerabilities
-        run: mage -v vuln     
+        run: mage -v vuln
+
+  generate-spec:
+    runs-on: ubuntu-latest
+    steps:
+    - name: fetch history
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19.2
+
+    - name: Install Mage
+      run: go install github.com/magefile/mage
+
+    - name: Update to latest spec
+      shell: bash
+      run: |
+        export PATH=$PATH:$(go env GOPATH)/bin
+        go install github.com/swaggo/swag/cmd/swag@v1.8.2
```

Comment:
the latest version swaggo does not work so I decided to use version 1.8.2

ref: https://github.com/swaggo/swag/issues/1249